scripts/telegraf.service

```diff
@@ -10,6 +10,7 @@ ExecStart=/usr/bin/telegraf -config /etc/telegraf/telegraf.conf -config-director
 ExecReload=/bin/kill -HUP $MAINPID
 Restart=on-failure
 RestartForceExitStatus=SIGPIPE
+RestartSec=5
```

Comment:
I have some concerns about this.

Restarting under normal conditions happens much more frequently than restarting under low memory conditions.  Delaying the restart will mean lost data in the common case for no reason.  It's not easy to weigh a potential benefit in an edge case vs a drawback in a much more common case, but I think this edge case is so small that it's not worth it.

Also, the choice of five seconds for the delay is completely arbitrary and not a general fix for restarting under low memory conditions for all machines.  It may be effective for the machines in the issue but it could be too short in other OOM conditions, such as when a lot of swap is involved.

Since this is a drawback under normal conditions and it's also not a general solution for dealing with out of memory state, I think it's better to leave it out.

We might suggest to the issue opener to use cgroup to limit memory usage of the user processes that might take up too much memory.  It's typically more effective to prevent the out of memory condition than try to clean up afterward.  If they don't want to do that, they can still add RestartSec=5 to telegraf.service after telegraf is installed.